backend/app/Repositories/EloquentWebsiteRepository.php

```diff
@@ -24,7 +24,7 @@ public function getById(int $id): Website
         return Website::findOrFail($id);
     }
 
-    public function getByTrackNumber(int $trackNumber): ?Website
+    public function getByTrackNumber(string $trackNumber): ?Website
```

Comment:
If you changed type that it means you broke some logic when it was used as int, doesn't it?